CMakeLists.txt

```diff
@@ -119,6 +119,12 @@ ELSE()
   ADD_LIBRARY(binaryen SHARED ${binaryen_SOURCES})
 ENDIF()
 TARGET_LINK_LIBRARIES(binaryen ${all_passes} wasm asmjs support)
+INSTALL(TARGETS binaryen DESTINATION lib)
+
+INSTALL(FILES src/binaryen-c.h DESTINATION include)
+INSTALL(FILES bin/wasm.js DESTINATION bin)
+INSTALL(FILES bin/binaryen.js DESTINATION bin)
+INSTALL(DIRECTORY src/js DESTINATION src)
```

Comment:
There are 5. But they all could be used by emscripten, so it seemed simpler to do this to ensure that new ones get picked up (and this command also creates the destination directory, so it seemed simpler than writing 6 other commands).